### PR TITLE
add some rules and checks

### PR DESCRIPTION
**Pull request by DmitryShonin**
add rules to network settings, implement mount options checking to 'sensitive' directories.

### CODE REVIEW

**Comment by DmitryShonin** [comment]
I know it's not necessary, but just in case.

**Comment by romanornr** [comment]
Ack

File: ubuntu-hardening.sh [hunk 1]
```diff
@@ -26,24 +26,56 @@ disable_compilers() {
...
+            No ) exit;;
+        esac
+    done
+
     # disable ICMP routing redirects
     sysctl -w net.ipv4.conf.all.accept_redirects=0
     sysctl -w net.ipv6.conf.all.accept_redirects=0
     sysctl -w net.ipv4.conf.all.send_redirects=0
-
+    sysctl -w net.ipv4.conf.default.accept_redirects = 0
```
Comment (DmitryShonin) [inline ubuntu-hardening.sh]: oh must be no spaces here

**Comment by romanornr** [comment]
Thanks

**Pull request merged**